### PR TITLE
add Credit Card Fraud Detection using Snowflake ML Functions

### PR DESCRIPTION
**Pull request by sfc-gh-kgovindarajan**

### CODE REVIEW

File: site/package-lock.json
Comment (jdanielmyers) [inline site/package-lock.json:1]: please remove from PR

File: site/sfguides/src/credit-card-fraud-detection-using-snowflake-ml/_shared_assets/Snowflake_SwitchRole.png
Comment (jdanielmyers) [inline site/sfguides/src/credit-card-fraud-detection-using-snowflake-ml/_shared_assets/Snowflake_SwitchRole.png:1]: please delete this file

File: site/sfguides/src/credit-card-fraud-detection-using-snowflake-ml/assets/SAMPLE.jpg
Comment (jdanielmyers) [inline site/sfguides/src/credit-card-fraud-detection-using-snowflake-ml/assets/SAMPLE.jpg:1]: please delete

**Review by jdanielmyers — CHANGES_REQUESTED** [review]

File: site/sfguides/src/credit-card-fraud-detection-using-snowflake-ml/credit-card-fraud-detection-using-snowflake-ml.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+How to build a Model using Snowflake ML Classification Function
+How to build a Streamlit app to detect and visualize the fraudulent transactions in Snowflake
+How to render a map and plot in the Streamlit app
+
+### What You'll Need
+- Snowflake account with a SYSADMIN role or a role that has CREATE SNOWFLAKE.ML.CLASSIFICATION privilege on the schema the model will be created
+- A GitHub Account
+- Intermediate knowledge of Snowsight, the UI interface of Snowflake
+
+# Setup
```
Comment (jdanielmyers) [inline site/sfguides/src/credit-card-fraud-detection-using-snowflake-ml/credit-card-fraud-detection-using-snowflake-ml.md]: broken syntax. Please render this file locally to ensure proper step structure.

**Review by jdanielmyers — CHANGES_REQUESTED** [review]

File: site/sfguides/src/credit-card-fraud-detection-using-snowflake-ml/credit-card-fraud-detection-using-snowflake-ml.md (hunk 1 above)
Comment (sfc-gh-kgovindarajan) [reply site/sfguides/src/credit-card-fraud-detection-using-snowflake-ml/credit-card-fraud-detection-using-snowflake-ml.md]: Thanks Daniel,  Just fixed that.. Please review when you get a chance.

**Pull request merged**